float64[6] stiffness